draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2337,13 +2465,13 @@ Fields MUST be chosen as follows:
 - CURVE\_ID: a human-readable representation of the target elliptic curve.
 
 - HASH\_ID: a human-readable representation of the hash function used in
-  hash\_to\_base ({{hashtobase}}).
+  hash\_to\_field ({{hashtofield}}).
 
-  If a suite uses an alternative hash\_to\_base function ({{hashtobase-alt}}),
+  If a suite uses an alternative hash\_to\_field function ({{hashtofield-expand-other}}),
   a short descriptive name MUST be chosen for that function using only the
   allowed characters listed above.
   That name MUST be appended to the HASH\_ID field, separated by a colon.
-  For example, a hash\_to\_base function based on KMAC128 {{SP.800-185}} might
+  For example, a hash\_to\_field function based on KMAC128 {{SP.800-185}} might
   use the short name "h2b/kmac128", and a reasonable value for the HASH\_ID field
   would be "SHA3:h2b/kmac128".
```

Comment:
Change to `h2f/kmac128` and `SHA3:h2f/kmac128`

Reply:
this is now gone because of edits to the Suites section.